build-logic/performance-testing/src/main/kotlin/gradlebuild/performance/PerformanceTestPlugin.kt

```diff
@@ -88,12 +87,12 @@ import javax.xml.parsers.DocumentBuilderFactory
 
 
 object Config {
-    const val performanceTestScenarioListFileName = "performance-tests/scenario-list.csv"
     const val performanceTestReportsDir = "performance-tests/report"
     const val performanceTestResultsJsonName = "perf-results.json"
     const val performanceTestResultsJson = "performance-tests/$performanceTestResultsJsonName"
     const val androidStudioVersion = "2021.1.1.19"
     val defaultAndroidStudioJvmArgs = listOf("-Xms256m", "-Xmx4096m")
+    const val defaultBaseline = "7.6-commit-0778ab5"
```

Comment:
Would it make sense to move the actual value to `gradle.properties` or maybe another file like `performance-baseline.txt` in the performance subdirectory? Changing this constant would basically mean that nearly everything would need to be rebuilt because of a build logic change.

Reply:
This is now in `gradle.properties`.